src/routes/controllers/productsControllers/getProductImage.js

```diff
@@ -0,0 +1,46 @@
+import Joi from '@hapi/joi';
+import mongoose from 'mongoose';
+import mongodb from 'mongodb';
+import Grid from 'gridfs-stream';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+const conn = mongoose.createConnection(process.env.DB_URI, {
+	useNewUrlParser: true,
+	useUnifiedTopology: true,
+});
+
+let gfs;
+conn.once('open', () => {
+	// init stream
+	gfs = Grid(conn.db, mongoose.mongo);
+	gfs.collection('product_images');
+});
+
+const getProductImage = (req, res) => {
+	try {
+		gfs.files.findOne({ filename: req.params.filename }, (err, file) => {
+			if (!file) return res.status(404).json({ msg: 'No file found' });
+
+			console.log(file);
+
+			// check if image
+			if (
+				file.contentType === 'image/jpeg' ||
+				file.contentType === 'image/png' ||
+				file.contentype === 'img/png'
```

Comment:
`file.contentype`? Is this needed?

Reply:
Yes, just to be sure we are only returning images incase unsupported files were uploaded

Reply:
Yes, but there is a type in `file.contentype`. Shouldn't it be `file.contentType` or file.contenttype?

Reply:
removed it. should be fixed now